frontend/public/components/conditions.tsx

```diff
@@ -5,7 +5,7 @@ import { Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
 
 export const Conditions: React.SFC<ConditionsProps> = ({conditions}) => {
-  const rows = _.map(conditions, condition => <div className="row" key={condition.type}>
+  const rows = _.map(conditions, condition => <div className="row" key={JSON.stringify(condition)}>
```

Comment:
Unfortunately, the conditions on `ClusterServiceVersions` do not follow standard Kube and have multiple of the same `type`, so we don't get a unique key for React.